backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/utils/SecretsWrapper.java

```diff
@@ -0,0 +1,20 @@
+package org.ovirt.engine.core.common.utils;
+
+public class SecretsWrapper<T> {
```

Comment:
I would better use a name like `SecretValue`. Because it says about a substance the class is presenting rather than a design pattern like "wrapper".

Reply:
Done.